.travis.yml

```diff
@@ -47,11 +47,13 @@ dist: trusty
 sudo: required
 addons:
    postgresql: "9.4"
-   apt:
-    packages:
-    - mysql-server-5.6
-    - mysql-client-core-5.6
-    - mysql-client-5.6
+   mysql: "5.7"
```

Comment:
In `invenio-acccess`, mysql also just openly specified: https://github.com/inveniosoftware/invenio-access/blob/master/.travis.yml#L42

@lnielsen At this point eg. Zenodo is running postgres 9.4, but shouldn't we pin specific versions of DBs/services just to have some guarantee of minimum requirements?